.bazelci/presubmit.yml

```diff
@@ -63,10 +63,15 @@ platforms:
     build_targets:
     - "--"
     - "..."
+    - "-//testdata:pkg_control_xz"
+    - "-//tests/docker:deb_image_with_dpkgs"
+    - "-//tests/docker:deb_image_with_dpkgs_test"
+    - "-//tests/docker:test_digest_output1"
     build_flags:
-    - "--extra_execution_platforms=@bazel_toolchains//configs/ubuntu16_04_clang/1.1:nosla_xenial_docker"
-    - "--host_platform=@bazel_toolchains//configs/ubuntu16_04_clang/1.1:nosla_xenial_docker"
-    - "--platforms=@bazel_toolchains//configs/ubuntu16_04_clang/1.1:nosla_xenial_docker"
+    - "--extra_execution_platforms=@bazel_toolchains//configs/ubuntu16_04_clang/1.2:nosla_xenial_docker"
+    - "--host_platform=@bazel_toolchains//configs/ubuntu16_04_clang/1.2:nosla_xenial_docker"
```

Comment:
You can probably use the latest docker_platform and toolchain alias here as well https://github.com/bazelbuild/bazel-toolchains/blob/master/.bazelci/presubmit.yml#L20

Reply:
fixed